pkg/apis/devices.harvesterhci.io/v1beta1/pcidevice.go

```diff
@@ -116,7 +120,9 @@ func (status *PCIDeviceStatus) Update(dev *pci.Device, hostname string) {
 	// Generate the ResourceName field, this is used by KubeVirt to schedule the VM to the node
 	status.ResourceName = resourceName(dev)
 	status.Description = description(dev)
-
+	if iommuGroup, err := device_manager.Handler.GetDeviceIOMMUGroup(pciBasePath, dev.Address); err == nil {
```

Comment:
Uh-oh, this will SIGSEGV without: https://github.com/kubevirt/kubevirt/blob/v0.54.0/pkg/virt-handler/device-manager/common.go#L198-L202

Reply:
https://github.com/harvester/pcidevices/pull/30#discussion_r1049598061 fixes it